### PR TITLE
Results Page - Final Shared Quiz Component

### PR DESCRIPTION
**Pull request by LuckyHariharan**
Hi, in this PR I setup draft1 of the results page. 

WIP Page: 
image
Figma Design: 
image

### CODE REVIEW

**Comment by LuckyHariharan** [comment]
Adjusting colour details, and trying to add that gray line. any feedback welcome @vithushan19 !

**Comment by jimip85** [comment]
Hey Lucky,

Everything seems to look good. If I were nitpicking I would say we have to adjust the navbar! But, if it's just a temp placeholder, it should be good to go!

**Review by jimip85 — COMMENTED** [review]
Hey Lucky,

Everything seems to look good. If I were nitpicking I would say we have to adjust the navbar! But, if it's just a temp placeholder, it should be good to go!

**Comment by LuckyHariharan** [comment]
> Hey Lucky,
> 
> Everything seems to look good. If I were nitpicking I would say we have to adjust the navbar! But, if it's just a temp placeholder, it should be good to go!

Oh for sure. Ty for pointing that out!

**Comment by vithushan19** [comment]
Check your page imports

**Comment by LuckyHariharan** [comment]
> 
> 
> Check your page imports

@vithushan19 ty for pointing this out. correcting the import fixed the vercel issue

File: pages/studentPortal/quizcomponent/results-page.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        </div>
+        <div className="font-bold mt-8 text-2xl text-black-600 mb-2">
+          The Skillify Blueprint
+        </div>
+      </div>
+      <div className="grid grid-cols-[1fr_3fr] md:grid-cols-[1fr_7fr]">
+        <div className="grid grid-cols-1 ml-8">
+          <NumberedCircles />
+        </div>
+        <div className="grid grid-cols-1 mx-4 mr-16  space-y-8 md:space-y-12  md:mt-2">
```
Comment (LuckyHariharan) [inline pages/studentPortal/quizcomponent/results-page.tsx:21]: check if grid is needed

**Review by vithushan19 — APPROVED** [review]
- move NumberedCircles into the components folder
- for the numbered circles, confirm that you understand what the relative and absolute styles are doing
- for the results page confirm that you understand what the grid syntax is doing

**Pull request merged**